tello_terrain_follow_node:
  ros__parameters:
    target_distance: 1.0
    kp: 0.7
    ki: 0.1
    kd: 0.05
    max_vel: 1.0
    min_vel: -1.0
    rel_alt_topic:  "/tello/relative_altitude"
    vertical_cmd_vel_topic: "/tello/vertical_cmd_vel"